packages/hooks/src/hooks/useAtomicBatchSupport.ts

```diff
@@ -0,0 +1,43 @@
+import { type AdaptedWallet } from '@reservoir0x/relay-sdk'
```

Comment:
Does it make sense to make this a part of the official relay hooks to export or is it better to keep it as an internal hook in the ui kit?

Reply:
We can move it to be an internal hook but figured it could be helpful for other devs implementing batch txs with the kit

Reply:
yeah, I would just prefer not to have incoming feature requests or having to maintain it for other devs.

Reply:
Yeah fair. Will move it